src/components/pages/NavBar.tsx

```diff
@@ -0,0 +1,29 @@
+import classNames from "classnames";
+import { NavLink } from "react-router-dom";
+
+export const Navbar = () => {
+  const getLinkClass = ({ isActive }: { isActive: boolean}) => classNames('navbar-item',{
+    'has-background-grey-lighter': isActive,
+  });
+
+  return (
+    <nav
+      data-cy="nav"
+      className="navbar is-fixed-top has-shadow"
+      role="navigation"
+      aria-label="main navigation"
+    >
+      <div className="container">
+        <div className="navbar-brand">
+          <NavLink to="/" className={({ isActive }) => getLinkClass({ isActive })}>
+            Home
+          </NavLink>
+
+          <NavLink to="/people" className={({ isActive }) => getLinkClass({ isActive })}>
+            People
+          </NavLink>
```

Comment:
The same issue is present here. You should use the `activeClassName` prop to conditionally style your `NavLink` based on whether it is active.